### DISCUSSION TITLE
Is it possible to use this for mobile--through react native?

### DISCUSSION THREAD

**Discussion by kenhuang1964** [robertknight/tesseract-wasm#103] [Q&A]
Is it possible to use this with react native?

**Comment by robertknight** [comment]
As far as I know React Native apps can't directly execute WebAssembly, which is a requirement for using this package. It would be possible to load the library in a WebView in an RN app, but that's probably going to be quite cumbersome to set up.

**Reply by kenhuang1964** [reply]
Got it, thanks!
